infor-design/enterprise
# 3587 - Fix inconsistencies of spacing on required fields

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixes the inconsistencies of spacing on required fields. It should just have a `7px` margin on top of the required text.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/3587

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

1.)

- Navigate to http://localhost:4000/components/modal/test-multiple-validation.html
- Open `Modal #1` button
- Tab out all the fields to trigger the error message
- Both fields should have the same `7px` spacing on top of the error message
- Test also on uplift

2.)
- Navigate to http://localhost:4000/components/modal/test-required-fields.html
- Open `Add Context` button
- Tab out searchfield to trigger the error message
- Error message should not overlap on searchfield
- Test also on uplift

3.)
- Navigate to http://localhost:4000/components/modal/test-validation-editor.html
- Open `Add Context` button
- Tab out all the fields to trigger the error message
- All fields should have the same `7px` spacing on top of the error message
- Error message should not far away on the editor
- Test also on uplift

4.) Found a related bug upon checking all the examples in modal component
- Navigate to http://localhost:4000/components/modal/test-body-scroll.html
- Scroll down to the bottom and open `Add Context` button
- Tab out all the fields to trigger the error message
- The last textarea should not too close now and has the same spacing too

**Included in this Pull Request**:
